### DISCUSSION TITLE
Touchscreen

### DISCUSSION THREAD

**Discussion by samand587** [beabel/radiowecker#69] [Fragen & Antworten / Q&A] [answered]
Ich habe eine 2,8"-Taste auf dem Display, die sich in der oberen rechten Ecke befindet. Wie kann ich das beheben?
![TACH](https://github.com/user-attachments/assets/b1b786b3-41b1-4541-bd84-e651fa38f938)

**Comment by beabel** [comment]
So richtig verstehe ich deine Frage nicht, jedoch vermute ich das das Display verdreht ist.

Versuche es mit diesem [Beitrag](https://github.com/beabel/radiowecker/discussions/53)

**Comment by samand587** [comment]
Hallo! Ich habe einen Sensor zum Einschalten des Radios in der oberen rechten Ecke. Ich habe es oben auf dem Foto markiert. Ich habe versucht, nach Ihren Empfehlungen Änderungen an der tft_display-Datei vorzunehmen, erhalte jedoch einen Fehler. Ich bitte Sie, Änderungen an der tft_display-Datei vorzunehmen und hier zu posten.   Danke!

**Comment by beabel** [comment]
Langsam.
Also wenn die Texte welche hier geschrieben werden so sinnfrei sind, kann man auch nicht korrekt helfen.
Einen Sensor im Display? Wäre mir neu.
Am einfachsten wäre mal die set rotation zu ändern, und dann schauen, wo dazu passend der Touch ist.
Es gibt hier noch jemanden, welcher ein gedrehtes Display bzw. gedrehten Touch hatte, auch er hat selbst versucht die umrechnung passend zu seiner "verdrehung" zu erstellen.
In dem gezeigten Stück Code werden die Touch Koordinaten umgerechnet, so das sie wieder passen.

**Comment by samand587** [comment]
Hallo! Ich möchte es klar schreiben, aber der Übersetzer funktioniert so. Ich habe einen Sensor zum Einschalten des Radios in der oberen rechten Ecke des Bildschirms. Ich wollte es selbst korrigieren , aber es fehlt an Erfahrung. Bitte nehmen Sie Änderungen an der tft_display-Datei für meinen Bildschirm vor(abgebildet).

**Reply by beabel** [reply]
versuche es damit: [Deepl](https://www.deepl.com/de/translator)

Damit kann man auch die Kommentare im Code übersetzen, und vielleicht damit auf eine Lösung kommen.

попробуйте это: [Deepl](https://www.deepl.com/de/translator)

С его помощью вы также можете перевести комментарии в коде и, возможно, найти решение.

**Comment by samand587** [comment]
Ich habe drei Tage lang versucht, die Skizze zu ändern. Ohne Erfolg! Wir müssen das Projekt aufgeben. Schade!

**Comment by fraeggle** [comment, marked as answer]
Wie @beabel bereits geschrieben hat ist vermutlich deine Rotataion falsch.
Einfach in der tft_display.ino im Abschnitt:
void setup_display() {.........................
--> von  
`tft.setRotation(3);`
auf   
`tft.setRotation(1);`

ändern.
